modules/ROOT/pages/custom-ingress-configuration.adoc

```diff
@@ -114,8 +114,6 @@ If you add the TLS section with a `secretName` attribute, you must ensure that y
 
 <7> These placeholder values are required for Kubernetes validation, but the actual values are not used by Runtime Fabric. 
 
-For a path-based routing ingress template, consult your ingress controller documentation and configure it accordingly. For an example using Nginx, see https://help.mulesoft.com/s/article/Path-based-Nginx-Ingress-Routing-Example-for-Runtime-Fabric-on-Self-Managed-Kubernetes[Path-based Nginx Ingress Routing Example for Runtime Fabric on Self-Managed Kubernetes].
```

Comment:
@digvijaysk19 @hegderohit89 can you add your clarification for the path based routing on line 113? 

Currently, states the following:

_<6> The `path` parameter value is replaced by the value you add in the *Path* field when you xref:configure-ingress-for-a-mule-application[configure ingress for a Mule application]. Note that `path` is not a placeholder._

Reply:
@digvijaysk19 can you please review and adjust if i have missed anything